aerogear/app-metrics-operator
# Add same standards applied in other projects as for example UPS oper

## Motivation
Apply the same standard used in the other projects as UPS Oper in order to add the Monitoring features easily. (https://issues.jboss.org/browse/AEROGEAR-9598)

## What
- Add GitHub files from standards ( same applied in all projects ) 
- Same Makefile structure of UPS Oper
- Same README structure of UPS Oper
- Add unit test, coveralls and CI as UPS Oper
- Add release process as UPS Oper
- Add e2e test as it was done for UPS Oper
- Improve APP NAMESPACE and OPER namespace validation in order to allow unit test and run it locally + unit tests to ensure that it is working as it was done for other projects. 

## Why
- Be able to easily create the monitor command and apply changes horizontally
- Follow the standards and make easier to keep maintained
- Add requirements to create the SOP's and Monitor features  

## Steps to verify
- **Check the README changes;** ( same ups oper structure )
- **Check that the CI working:**
- **Check that the Jenkins is working here with the e2e test as well now**.

<img width="1026" alt="Screenshot 2019-07-15 at 16 25 52" src="https://user-images.githubusercontent.com/7708031/61242917-41b85380-a71d-11e9-80fa-859ffeed5138.png">


## Checklist:

## Progress
